examples/magic_parameter_sweep.py

```diff
@@ -16,28 +16,23 @@
 # In this example, because we have already imported napari anyway, it doesn't
 # really matter. But this syntax would let you specify that a parameter is a
 # napari object type without actually importing or depending on napari.
+# We also use the `Annotated` type to pass additional information that can be used
+# to aid widget generation.
```

Comment:
probably should specify that the keys of these dicts are magicgui widget kwargs, so they know what's available

Reply:
good point

Reply:
is there a magicgui link i can put in?

Reply:
sure, just say "keywords must be valid for the corresponding magicgui widget type", and link to https://napari.org/magicgui/api/widgets.html